qiita_ware/ebi.py

```diff
@@ -1060,6 +1062,7 @@ def _generate_demultiplexed_fastq_demux(self, mtime):
                 raise EBISubmissionError(error_msg)
             for s, i in to_per_sample_ascii(demux_fh,
                                             self.prep_template.keys()):
+                s = s.decode('ascii')
```

Comment:
Any chance there will be characters in the data that could make this fail or produce /x000 type encodings that would cause problems downstream?

Reply:
Anything is possible but IMOO almost impossible as these are sample names and these are fields that are really restricted so we _should_ be fine.

Reply:
Sounds reasonable.